### PR TITLE
[Views] Handle when the github name is None

### PR DESCRIPTION
**Pull request by segiddins**

### CODE REVIEW

File: sotu/views.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def callback(request):
     username = user['login']
 
     name = user.get('name', username)
-    if len(name) == 0:
+    if name == None || len(name) == 0:
```
Comment (k0nserv) [inline sotu/views.py]: `name is None` is more pythonic i believe

File: sotu/views.py [hunk 2]
```diff
@@ -112,7 +112,7 @@ def callback(request):
     username = user['login']
 
     name = user.get('name', username)
-    if len(name) == 0:
+    if name is None || len(name) == 0:
```
Comment (k0nserv) [inline sotu/views.py]: Also `||` is not valid python, use `or` instead

Comment (segiddins) [inline sotu/views.py]: you sure? The build seems to be passing

Comment (k0nserv) [inline sotu/views.py]: Local test fails, what version of python does this run?

Comment (k0nserv) [inline sotu/views.py]: local tests as in me testing it in the REPL

Comment (k0nserv) [inline sotu/views.py]: Under 3.5 too
image

**Comment by k0nserv** [comment]
Cool, looks 🆗  to merge on green CI

**Pull request merged**